pkg/pillar/nireconciler/linux_acl.go

```diff
@@ -515,6 +515,59 @@ func (r *LinuxNIReconciler) withFlowlog() bool {
 	return false
 }
 
+func (r *LinuxNIReconciler) doACLsAllowAllTraffic() (
+	allIpv4TrafficAllowed, allIpv6TrafficAllowed bool) {
+	allIpv4TrafficAllowed = true
+	allIpv6TrafficAllowed = true
+	for _, app := range r.apps {
+		if app.deleted {
+			continue
+		}
+		for _, adapter := range app.config.AppNetAdapterList {
+			var hasAllowAllRuleIPv4, hasAllowAllRuleIPv6 bool
+			for _, ace := range adapter.ACLs {
+				// Default action (if not specified) is to allow traffic to continue.
+				allowRule := true
+				for _, action := range ace.Actions {
+					if action.Drop || action.Limit {
+						// Let's keep this simple and not try to distinguish
+						// between IPv4 and IPv6 DROP/LIMIT rules.
```

Comment:
Make sense. At the same time, we still distinguish between ipv4 and ipv6 when checking the allow rules. Is there a specific reason for that?

Reply:
Got it, thanks!